backend/src/main/java/gov/cdc/usds/simplereport/validators/PatientBulkUploadFileValidator.java

```diff
@@ -0,0 +1,112 @@
+import static gov.cdc.usds.simplereport.validators.CsvValidatorUtils.validateEmail;
+import static gov.cdc.usds.simplereport.validators.CsvValidatorUtils.validateEthnicity;
+import static gov.cdc.usds.simplereport.validators.CsvValidatorUtils.validatePhoneNumber;
+import static gov.cdc.usds.simplereport.validators.CsvValidatorUtils.validatePhoneNumberType;
+import static gov.cdc.usds.simplereport.validators.CsvValidatorUtils.validateRace;
+import static gov.cdc.usds.simplereport.validators.CsvValidatorUtils.validateRole;
+import static gov.cdc.usds.simplereport.validators.CsvValidatorUtils.validateState;
+import static gov.cdc.usds.simplereport.validators.CsvValidatorUtils.validateYesNoAnswer;
+import static gov.cdc.usds.simplereport.validators.CsvValidatorUtils.validateZipCode;
+
+import com.fasterxml.jackson.databind.MappingIterator;
+import gov.cdc.usds.simplereport.service.model.reportstream.FeedbackMessage;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import org.springframework.stereotype.Component;
+
+@Component
+public class PatientBulkUploadFileValidator {
+
+  public List<FeedbackMessage> validate(InputStream csvStream) {
+    final MappingIterator<Map<String, String>> valueIterator = getIteratorForCsv(csvStream);
+
+    if (!valueIterator.hasNext()) {
+      throw new IllegalArgumentException("Empty or invalid CSV submitted");
+    }
+
+    List<FeedbackMessage> errors = new ArrayList<>();
+
+    while (valueIterator.hasNext() && errors.isEmpty()) {
+      final Map<String, String> row = getNextRow(valueIterator);
+
+      ValueOrError firstName = getValue(row, "FirstName", true);
+      ValueOrError lastName = getValue(row, "LastName", true);
+      ValueOrError middleName = getValue(row, "MiddleName", false);
+      ValueOrError suffix = getValue(row, "Suffix", false);
+      ValueOrError race = getValue(row, "Race", true);
+      ValueOrError dob = getValue(row, "DOB", true);
+      ValueOrError biologicalSex = getValue(row, "biologicalSex", true);
+      ValueOrError ethnicity = getValue(row, "Ethnicity", true);
+      ValueOrError street = getValue(row, "Street", true);
+      ValueOrError street2 = getValue(row, "Street2", false);
+      ValueOrError city = getValue(row, "City", false);
+      ValueOrError county = getValue(row, "County", false);
+      ValueOrError state = getValue(row, "State", true);
+      ValueOrError zipCode = getValue(row, "ZipCode", true);
+      ValueOrError phoneNumber = getValue(row, "PhoneNumber", true);
+      ValueOrError phoneNumberType = getValue(row, "PhoneNumberType", false);
+      ValueOrError employedInHealthcare = getValue(row, "employedInHealthcare", true);
+      ValueOrError residentCongregateSetting = getValue(row, "residentCongregateSetting", true);
+      ValueOrError role = getValue(row, "role", false);
+      ValueOrError email = getValue(row, "email", false);
```

Comment:
I know we've made individual values case-insensitive, but what about the CSV headers? According to the bulk patient upload doc, these two headers are capitalized:

<img width="649" alt="Screen Shot 2022-10-27 at 1 51 19 PM" src="https://user-images.githubusercontent.com/27730981/198362867-1b29adc8-558a-4eea-a847-e806baf93650.png">

Reply:
Yes, good catch! I just discovered this yesterday in the course of trying to use the validator :) Updated them both to be capitalized.